cfpb/cfgov-refresh
# owning-a-home: Adds rel="noopener noreferrer" to targets

Addresses security issue in [GHE]/CFGOV/platform/issues/2522

## Changes

- Adds `rel="noopener noreferrer"` to "_blank" href targets.
- Fixes issue in social media molecule where "_blank" was not quoted.

## Testing

1. Ensure `OAH_FORM_EXPLAINERS` flag is true.
2. Visit /owning-a-home/loan-estimate/ (e.g.) and inspect the links in the right-hand sidebar that open in a new window and see that they have `rel="noopener noreferrer" in the link.
